### PR TITLE
Feature/color_by_change_options

### PR DESCRIPTION
**Pull request by gmalinve**

### CODE REVIEW

File: pyaedt/hfss3dlayout.py [hunk 1]
```diff
@@ -2202,3 +2202,50 @@ def get_dcir_element_data_via(self, setup_name):
...
                 df.merge(df_tmp, left_index=True, right_index=True, how="outer")
         return df
+
+    @pyaedt_function_handler
+    def show_extent(self, show=True):
+        """Show or hide extent in a HFSS3dLayout design.
+
+        Parameters
+        ----------
+        show: bool, optional
```
Comment (MaxJPRey) [inline pyaedt/hfss3dlayout.py:2212]: This should fix the documentation failure.
```suggestion
        show : bool, optional
```

File: pyaedt/hfss3dlayout.py [hunk 2]
```diff
@@ -2202,3 +2202,50 @@ def get_dcir_element_data_via(self, setup_name):
...
+        Parameters
+        ----------
+        color_by_net: bool, optional
+            Whether visualize color by net or by layer.
+            The default value is ``True``, which means color by net.
+
+        Returns
+        -------
+        bool
+            ``True`` is successful, ``False`` if it fails.
```
Comment (MaxJPRey) [inline pyaedt/hfss3dlayout.py:2243]:
```suggestion
            ``True`` if successful, ``False`` if it fails.
```

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Review by Samuelopez-ansys — APPROVED** [review]
@gmalinve Could you add an example to the new methods? "show_entent" and "change_options"

**Pull request merged**